modules/system/assets/ui/js/tab.js

```diff
@@ -84,6 +84,10 @@
         })
 
         this.updateClasses()
+
+        if (this.$tabsContainer.is('[data-linkable]') && location.hash) {
```

Comment:
@Rike-cz would be better to switch those, micro speed optimization: `if (location.hash && this.$tabsContainer.is('[data-linkable]') ) { `

Wouldn't it be better to use has `hasClass` instead of `is`? just to check if  `tabsContainer` has`data-linkable` class?

Reply:
@w20k `data-linkable` is an attribute, not a class.

Reply:
OK, so I will swap conditions only, no problem. But bellow I see "This branch has conflicts that must be resolved". Whats wrong there?

Reply:
Oooo - this excites me!

Reply:
Fixed it for you @Rike-cz